catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/RemoveCommand.java

```diff
@@ -81,53 +91,50 @@ private Object executeRemoveFromCache() throws Exception {
     }
 
     private Object executeRemoveFromStore() throws Exception {
+
         CatalogFacade catalogProvider = getCatalog();
 
+        int batchCount = 1;
```

Comment:
Where is `batchCount` used?

Reply:
It's used for logging the current batch in debug. Not really important. I'll keep it in if there if I don't see any objections.